### DISCUSSION TITLE
Copy PlainText

### DISCUSSION THREAD

**Discussion by algominded** [erosman/support#326] [Q&A]
Hi Erosman,
This might not be the appropriate place to post this, but I'll take my chances.
I updated the Copy PlainText extension today on Firefox and for some reason, it disappeared from the contextual menu.
By contextual menu, I mean the menu that appears when you right-click on any page in the browser.
Tried uninstalling/reinstalling it and it still hasn't worked the way it used to.

Any ideas on how to fix this?

Thank you

PS.: Feel free to delete this discussion if it's in the wrong place. Thought of doing it myself but seems that option is not available here. In hindsight it makes more sense for this question to be in the issues tab.

**Comment by erosman** [comment]
Hi algominded 

No problem with posting questions.

Which Firefox and which OS?
Are you testing it on private windows?
Does the Toolbar icon appear?
Try uninstalling then restart Firefox and then instal again and see what happens.
